CHANGELOG.md

```diff
@@ -1,7 +1,46 @@
 # Changelog
 
+## [v0.59.6]
+
+### Fixed
+
+- Fix regexp on `hasteImpl` ([bcd1e2](https://github.com/facebook/react-native/commit/bcd1e23) by [@CaptainNic](https://github.com/CaptainNic))
+- Fix sparse array handling in `EventEmitter#listeners()` ([f68dc8](https://github.com/facebook/react-native/commit/f68dc8) by [@ide](https://github.com/ide))
+- Fix **VirtualizedList** to call `_updateViewableItems` immediately ([08141e](https://github.com/facebook/react-native/commit/08141ef1559) by [@sahrens](https://github.com/sahrens))
+- Fix metro websocket reconnect logic ([c28676](https://github.com/facebook/react-native/commit/c286769d28c) by [@rickhanlonii](https://github.com/rickhanlonii))
```

Comment:
This is iOS only, and to users it probably doesnt matter about websocket but instead maybe mention the bundler after closing?